src/sbvr-api/translations.ts

```diff
@@ -208,6 +208,11 @@ export const translateAbstractSqlModel = (
 			const { $toResource, ...definition } = translationDefinition;
 			const hasToResource = typeof $toResource === 'string';
 			if (hasToResource) {
+				if ($toResource.includes('$')) {
```

Comment:
Is this the plain user defined value or is it pre-processed earlier?
Afaict all `$toResource` occurrencies that we atm have are do target the next model and thus include a `$` in them
eg: ```$toResource: `application$${toVersion}`,```
See: https://github.com/search?q=repo%3Abalena-io%2Fopen-balena-api%20%24toResource&type=code